docs/tldr/pages/linux/tmt-run.md

```diff
@@ -0,0 +1,37 @@
+# tmt run
+
+> Execute tmt test steps. By default, all steps are run.
+> See also: `tmt`, `tmt-try`.
+> More information: <https://tmt.readthedocs.io/en/stable/overview.html#run>.
+
+- Run all test steps for each plan:
+
+`tmt run`
+
+- Run selected plans and tests:
+
+`tmt run plan -n {{plan name}} test -n {{test name}}`
```

Comment:
```suggestion
`tmt run plan -n {{/plan/name}} test -n {{/test/name}}`
```

Looks like two arguments, would have to be quoted. Let's use slashes to make it more resemble actual `fmf` names?